docs/usage.rst

```diff
@@ -2,8 +2,353 @@
 Usage
 =====
 
-To use phenotrex in a project::
+The following quick start example shows training, evaluation and inference with phenotrex models
+using the phenotrex command line interface. For further information on flags and parameters used
+in this tutorial, please consult the relevant CLI documentation available via
+``phenotrex <command> --help``.
 
-    from phenotrex.io import ...  #  file I/O
-    from phenotrex.ml import ...  # classifiers and training/CV functionality
-    from phenotrex.util import ...  # plotting and util functions
+To illustrate phenotrex's capabilities, a small dataset of genome assemblies will be used.
+To download all required data, run:
+
+.. code-block:: console
+
+    $ curl -OL http://fileshare.csb.univie.ac.at/phenotrex/tutorial_data.tar
+    $ tar -xf tutorial_data.tar
+    $ cd tutorial_data
+
+To then install phenotrex, including its capability for extracting features from FASTA files,
+in a new virtual environment:
+
+.. code-block:: console
+
+    $ python3 -m venv env
+    $ source ./env/bin/activate
+    $ pip install phenotrex[fasta]
+
+
+Creation of Phenotrex Input Features
+------------------------------------
+Phenotrex operates on presence/absence patterns of `eggNOG`_ cluster IDs in the passed genome.
+If a DNA FASTA file is passed to phenotrex,
+`Prodigal`_ is first used to find protein sequences - this step is skipped if a protein FASTA file
+is passed instead. To then find eggNOG cluster IDs from protein sequences, `deepnog`_ is utilized.
```

Comment:
```suggestion
is passed instead. To then find eggNOG cluster IDs from protein sequences, `deepnog`_ is used.
```
"utilize" has a slight connotation of "abuse"

Reply:
TIL!